combination_test/test_assert.sh

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+cleanup() {
+	rm -f cmp out
+}
+
+assert() {
+	printf '%-30s:' "\"$1\""
+	# exit status
+	echo -n -e "$1" | bash >cmp 2>&-
```

Shellcheck (0.10.0)

[warning] Redirecting to/from command name instead of file. Did you want pipes/xargs (or quote to ignore)?

(SC2238)